app/views/shared/_error_messages.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if @user.errors.any? %>
+  <div id="error_explanation">
+    <div class="alert alert-danger">
+      <%= t ".the_form_contains" %>
+      <%= pluralize @user.errors.count, "error" %>.
```

Comment:
count vs length khac j nhau e

Reply:
count sẽ load nhanh hơn length ạ
- count thì đếm số lượng
- length cũng đếm số lượng nhưng sẽ load toàn bộ record